traffic_ops/traffic_ops_golang/urisigning.go

```diff
@@ -65,12 +65,11 @@ func getURIsignkeysHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	cluster, err := riaksvc.StartCluster(inf.Tx.Tx, inf.Config.RiakAuthOptions)
```

Comment:
So I pulled this PR down to check it out and found that the `StartCluster` and `StopCluster` are no longer used. With those removed, `GetRiakClusterTx` also becomes unused. Then there is `RiakServersToCluster` which is only used by the `riaksvc.Ping` function. I think we should have `Ping` use the `WithCluster` function to reuse the shared cluster rather than have the `Ping` function be the only function that actually recreates the `RiakCluster` every time. Then we are only left with functions that properly reuse the shared `RiakCluster`, making it harder for new code to do the wrong thing.

So to be clear I'm proposing:
1. remove `StartCluster`, `StopCluster`, `GetRiakClusterTx`, and `RiakServersToCluster`.
2. change `riaksvc.Ping` to use the `WithCluster` function rather than `RiakServersToCluster`
3. change `GetRiakServers` to `getRiakServers` (make it private), since it will only be used by `GetPooledClusterTx` at that point.

Reply:
The ping is designed to check individual machines in the cluster.  So it has to iterate on each machine in the server list, put it into its own cluster, then ping it.  Otherwise there's no way to individually determine health without going through the error logs directly.